linkml/generators/docgen/type.md.jinja2

```diff
@@ -1,8 +1,10 @@
-# {{ gen.name(element) }}
+# Type: {{ gen.name(element) }}
 
-{{ element.description }}
+{% if element.description %}
+_{{ element.description }}_
+{% endif %}
 
-URI: {{ gen.uri(element) }}
+URI: [{{ gen.uri(element) }}]({{ gen.uri(element) }})
```

Comment:
why not use `docgen.link(...)`?

Reply:
Switched to using `docgen.uri_link()` instead